### PR TITLE
Add example markup and styles for Course Hub name-pronounciation fields.

### PR DESCRIPTION
**Pull request by adamfranco**
Hi Zeb,

I'm working on adding name-pronunciation fields (name as spoken, recording, and phonetic spelling) to the Hub and put together these tweaks for the theme.

One issue: When printing, if full height to include the pronunciation is used, then only two profiles fit on each page. However, if the height is restricted to fit four profiles on a page (as currently in this PR, then some content may get hidden behind subsequent photos:
Screen Shot 2020-04-22 at 11 38 50 AM

Also, I added example markup that I'm generating in Drupal, but couldn't actually test it in the static version because of this error when building the frontend:

```
Error parsing twig template src/templates/course-hub-layout.twig:
TypeError: this.parentTemplate.getBlock is not a function
    at Twig.Template.getBlock (/home/afranco/private_html/drupal/midd-frontend/node_modules/twig/twig.js:1716:35)
    at Twig.ParseState.getBlock (/home/afranco/private_html/drupal/midd-frontend/node_modules/twig/twig.js:1462:29)
    at Twig.ParseState.block (/home/afranco/private_html/drupal/midd-frontend/node_modules/twig/twig.js:4580:25)
    at /home/afranco/private_html/drupal/midd-frontend/node_modules/twig/twig.js:2626:38
    at Twig.Thenable.resolvedThen [as then] (/home/afranco/private_html/drupal/midd-frontend/node_modules/twig/twig.js:8734:35)
    at Twig.ParseState.parse (/home/afranco/private_html/drupal/midd-frontend/node_modules/twig/twig.js:2623:56)
    at /home/afranco/private_html/drupal/midd-frontend/node_modules/twig/twig.js:3099:40
    at next (/home/afranco/private_html/drupal/midd-frontend/node_modules/twig/twig.js:8958:16)
    at Object.Twig.async.forEach (/home/afranco/private_html/drupal/midd-frontend/node_modules/twig/twig.js:8967:12)
    at Twig.ParseState.<anonymous> (/home/afranco/private_html/drupal/midd-frontend/node_modules/twig/twig.js:3082:25)
```

I'm open to suggestions, modifications, or other directions for this change, but the first commit seems to be the minimal change needed for basic operation.

### CODE REVIEW

**Comment by zebapy** [comment]
Is switching to row layout (if we can maintain 4 items per page) an option? That may allow us to remove the fixed height, which is always problematic.

**Comment by adamfranco** [comment]
Row layout would be totally fine.

**Comment by zebapy** [comment]
Is this something you need today? I am off tomorrow so I can try to prioritize this if needed.

**Comment by adamfranco** [comment]
This can wait until until next week. I'm just getting the UI in place for more user-testing this week.

File: src/templates/partials/coursehub-profile.twig [hunk 1]
```diff
@@ -9,6 +9,23 @@
   <p class="f3 mb-3">{{text}}</p>
 
   <dl class="f2">
+    <div class="mb-2">
+      <dt class="d-block d-sm-inline text-bold">Pronounciation:</dt>
+      <dd class="d-inline">
+        <div class="name-spoken">" Jonathan Doe-Smith "</div>
```
Comment (zebapy) [inline src/templates/partials/coursehub-profile.twig:15]: @adamfranco Are these classes like `name-spoken` selected by any css/js in D8?

Comment (adamfranco) [reply src/templates/partials/coursehub-profile.twig:15]: Nope, I hadn't referenced them yet. They can be removed or changed.

**Comment by zebapy** [comment]
Screen Shot 2020-04-23 at 3 41 13 PM

if we want to reduce the audio controls to a smaller UI and perhaps even just a "Play" button, I could adjust our already custom audio player. This may not be needed but wanted to bring it up. 

I'm working on figuring out an arrangement of the meta info for a row layout. I'm not sure if I'll wrap it up today though.

**Pull request merged**